### ISSUE TITLE
Can't change user's contact email address

### ISSUE THREAD

**Issue by pini-gh** [haiwen/seahub#4740]
I'm aware of the answer for issue #340. But setting another email address as the user's reference ID doesn't update their contact email address.
BTW changing their contact email address should be left to the user, not the admin.

**Comment by mydogspies** [comment]
Yes, I agree on this. One can not even change the default settings for me@example.com. Also, why I as admin can not simply add a new user with admin privileges beats me? That must simply be up to me if I want to hand out admin accounts, or? Seems to my as such a basic function.
If I was trying to hack the server I would definitely try with me@example.com to save myself time. ^^
